environments/base/default-config.libsonnet

```diff
@@ -3,7 +3,7 @@
 
   name: 'observatorium-xyz',
   namespace: 'observatorium',
-  thanosVersion: 'master-2020-03-24-4bd19b16',
+  thanosVersion: 'v0.12.0-rc.0',
```

Comment:
Do we need to bump to the release candidate in order to get one a memcached feature? Otherwise I would prefer to leave the version bump to a follow up to keep this PR clean and the changeset more minimal. It also means we can roll back changes independently.

Reply:
I need `--experimental.enable-index-cache-postings-compression` flag for these changes.
I'm going to create a separate PR to bump version after `v0.12.0-rc1` and then merge this one.

Reply:
Will update this one after merging https://github.com/observatorium/configuration/pull/245